docs/evm_compatibility/overview.md

```diff
@@ -2,27 +2,51 @@
 title: EVM Compatibility Overview
 ---
 
-Neon EVM is designed to be compatible with the [Ethereum Virtual Machine (EVM)](https://ethereum.org/en/developers/docs/evm/) to provide a as seamless of a developer experience as possible, with some notable differences and considerations. 
+TL;DR
```

Comment:
@BoogVAr -- when you are better, can we connect over where the existing item fails to support the devs pls?
It is on the backlog to update this item to provide better support. I have improved the current structure and wording, but it may be that there is missing information also?